### PR TITLE
Updated tag in resource schema

### PR DESCRIPTION
**Pull request by richardycao**
*Issue #, if available:*

*Description of changes: Update tag in schema*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-ssm-maintenancewindow/aws-ssm-maintenancewindow.json [hunk 1]
```diff
@@ -75,10 +75,16 @@
             "additionalProperties": false,
             "properties": {
                 "Key": {
-                    "type": "string"
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 128,
+                    "pattern": "^([\\p{L}\\p{Z}\\p{N}_.:/=+\\-@]*)$"
```
Comment (feleyu) [inline aws-ssm-maintenancewindow/aws-ssm-maintenancewindow.json:81]: Would we be able to address the constraint that tag key cannot be prefixed with "aws:" in the pattern? Those tags will override our system tags.

File: aws-ssm-maintenancewindow/aws-ssm-maintenancewindow.json [hunk 2]
```diff
@@ -75,10 +75,16 @@
...
                 },
                 "Value": {
-                    "type": "string"
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 256,
+                    "pattern": "^([\\p{L}\\p{Z}\\p{N}_.:/=+\\-@]*)$"
```
Comment (feleyu) [inline aws-ssm-maintenancewindow/aws-ssm-maintenancewindow.json:87]: Same here. Reference: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html#cfn-resource-tags-key

File: aws-ssm-maintenancewindow/aws-ssm-maintenancewindow.json (hunk 1 above)
Comment (richardycao) [reply aws-ssm-maintenancewindow/aws-ssm-maintenancewindow.json]: will address this. Thanks!

**Comment by nikita-zolotykh** [comment]
> Double backslashes on the regex should be removed.

Ignore this. Realized the double backslash is to escape special characters in JSON. The resulting regex expression should be of correct format.

**Pull request merged**